src/main/markdown/eco/add-lib.md

```diff
@@ -0,0 +1,26 @@
+# Update Information
+
+We will add or update information on request. 
+
+## Request for adding Libraries to the Eco System site
```

Comment:
Maybe generalize "library" to "dependency", so we can discuss frameworks, plugins, etc?

Reply:
Good point. I'll switch from **library** to ** dependency** in general ... 

Done